build.gradle

```diff
@@ -69,6 +69,8 @@ task compileVersion (type: JavaCompile) {
     }
     sourceCompatibility = 1.8
     targetCompatibility = 1.8
+    compileJava.options.encoding = "UTF-8"
```

Comment:
I’d prefer this to be more global than just for this task, i.e.
```
tasks.withType(JavaCompile) {
  options.encoding = "UTF-8"
}
```
at top-level somewhere.

Reply:
I moved the definition to the same level where the sourceCompatibility level is defined globally, to keep them the same.

Reply:
Can you please use the way I suggested? This way it will always be set correctly for all java compilation tasks.

Reply:
tasks.withType(JavaCompile) {
    options.encoding = "UTF-8"
}

tasks.withType(Javadoc) {
    options.encoding = "UTF-8"
}

Reply:
I personally wouldn’t have bothered with the javadoc but that’s fine. :)